examples/jsm/renderers/nodes/accessors/CameraNode.js

```diff
@@ -0,0 +1,96 @@
+import Node from '../core/Node.js';
+import Vector3Node from '../inputs/Vector3Node.js';
+import Matrix4Node from '../inputs/Matrix4Node.js';
+import { NodeUpdateType } from '../core/constants.js';
+
+class CameraNode extends Node {
+
+	static POSITION = 'position';
```

Comment:
Sidenote: The default JavaScript parser of ESLint (Espree) does currently not support class fields syntax. Hence, the linter reports a parsing error:

> Unexpected token =

The only way to fix this right now is to use a different parser like `@babel/eslint-parser`. However, this will require an additional dev dependency and an enhancement of the `eslintConfig` section in `package.json`.

@mrdoob Unless we don't start using class fields in the core, I recommend to do nothing for now. This issue should hopefully be resolved when class fields are not experimental anymore.